infrastructure/application/utils/generateTeamSecrets.ts

```diff
@@ -19,21 +19,30 @@ const name = (name: string) => name.replace(regex, "_").toUpperCase();
  */
 const value = (value: string) => value.replace(regex, "-").toLowerCase();
 
-export const generateTeamSecrets = (config: pulumi.Config): awsx.ecs.KeyValuePair[] => {
+export const generateTeamSecrets = (
+  config: pulumi.Config,
+  env: String
+): awsx.ecs.KeyValuePair[] => {
   const secrets: awsx.ecs.KeyValuePair[] = [];
-  teams.forEach(team => {
+  teams.forEach((team) => {
     secrets.push({
       name: `GOV_UK_PAY_TOKEN_${name(team.name)}`,
-      value: config.require(`gov-uk-pay-token-${value(team.name)}`),
+      value:
+        env === "sandbox"
+          ? "sandbox"
+          : config.require(`gov-uk-pay-token-${value(team.name)}`),
     });
-    team.uniformInstances?.forEach(instance => {
+    team.uniformInstances?.forEach((instance) => {
       secrets.push({
         name: `UNIFORM_CLIENT_${name(instance)}`,
-        value: config.require(`uniform-client-${value(instance)}`),
+        value:
+          env === "sandbox"
+            ? "sandbox"
+            : config.require(`uniform-client-${value(instance)}`),
```

Comment:
I've added this ternary operator / conditional here just so that we don't have to set up pulumi configs for uniform secrets for each team on sandbox, as sandbox isn't going to need them but otherwise they'd be required in order to call `pulumi up`.